doc/CMakeLists.txt

```diff
@@ -29,6 +29,49 @@ configure_file(ext/traffic-server.cmake.in.py ext/traffic-server.py)
 configure_file(conf.cmake.in.py conf.py)
 configure_file(manpages.cmake.in.py manpages.py)
 
+# copy static directory to build
+add_custom_target(
+  generate_docs_setup
+  COMMENT "Copy static files to build directory"
+  COMMAND ${CMAKE_COMMAND} -E copy_directory
+          ${CMAKE_CURRENT_SOURCE_DIR}/static
+          ${CMAKE_CURRENT_BINARY_DIR}/static
+)
+
+# generate svg files from uml files
+set(UML_FILES
+  uml/JsonRPCManager.uml
+  uml/TLS-Bridge-Messages.uml
+  uml/TLS-Bridge-Plugin.uml
+  uml/hdr-heap-class.plantuml
+  uml/hdr-heap-str-alloc.plantuml
+  uml/host-resolve.plantuml
+  uml/l4-basic-sequence.uml
+  uml/l4-example-cdn-layout.uml
+  uml/l4-pre-warming-overview.uml
+  uml/l4-sni-routing-seq.uml
+  uml/l4-tcp-routing.uml
+  uml/traffic_ctl-class-diagram.uml
+  uml/url_rewrite.plantuml
+)
```

Comment:
Can we glob these? That should make things a bit easier for people adding these.

Reply:
We can glob them, but then a user will have to know to manually re-run cmake if they add a new file to the directory.  OTOH, adding a file to this list will auto generate cmake and then continue the build.  Its a trade off.

cmake doesn't support pattern matching that makefiles can do because other builds systems (ninja, Xcode, visual studios) don't have a similar feature.

Reply:
I see. Let's leave it as explicit then as you did for this patch. Thanks.